openapi/azure.openai.embeddings/Module.md

```diff
@@ -0,0 +1,67 @@
+## Overview
+This is a generated connector from [Azure OpenAI Embeddings API v2023-03-15-preview](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference#embeddings/) OpenAPI specification.
+
+The Azure Azure OpenAI Service REST API Embeddings Endpoint will generate one a vector representation of a given input that can be easily consumed by machine learning models and other algorithms.
+
+## Prerequisites
+- Create an [Azure](https://azure.microsoft.com/en-us/features/azure-portal/) account
+- Create an [Azure OpenAI resource with a model deployed](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/create-resource)
```

Comment:
Shall we list deploying a model as a different step? Also can we use numbers?

Reply:
Other Azure connectors have not used numbers for this section. So better not to I think. I'll have model creation in a separate step

Reply:
Using numbers for steps came by Praneesha. Can you check with her?

Reply:
Ack